docker/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM node:21.7.3-slim
+
+USER root
+RUN apt-get update && apt-get install -y wget
+
+# Set a non-root user to run the container
+RUN useradd --create-home --shell /bin/bash mesh
+
+USER mesh
+
+WORKDIR /app
+
+COPY package.json ./
+
+RUN npm i --only=production
+
+EXPOSE 4000
+
+ENTRYPOINT ["./node_modules/.bin/mesh-serve"]
```

I recommend using [`dumb-init`](https://github.com/Yelp/dumb-init) for proper signal propagation. (Install it with apt first)

```suggestion
ENTRYPOINT ["dumb-init", "./node_modules/.bin/mesh-serve"]
```